src/main/js/dynamic/JobCustomMessageModal.js

```diff
@@ -29,10 +30,8 @@ class JobCustomMessageModal extends Component {
         this.handleHide = this.handleHide.bind(this);
     }
 
-    createCustomMessageFieldModel(jobFieldModel) {
-        const fieldModel = jobFieldModel.fieldModels.find(model => model.descriptorName === this.props.channelDescriptorName);
-        let newModel = FieldModelUtilities.createEmptyFieldModel([TOPIC_ID, MESSAGE_ID], CONTEXT_TYPE.DISTRIBUTION, this.props.channelDescriptorName);
-        newModel = FieldModelUtilities.combineFieldModels(newModel, fieldModel);
+    createCustomMessageFieldModel() {
+        let newModel = FieldModelUtilities.createEmptyFieldModel([TOPIC_ID, MESSAGE_ID], CONTEXT_TYPE.DISTRIBUTION, `${this.props.channelDescriptorName}-CUSTOM_MESSAGE`);
```

Comment:
Why would we append -CUSTOM_MESSAGE here?

Reply:
we're adding a new field model to the JobModel.  A JobModel is a collection of field models for each descriptor.